readme.md

```diff
@@ -1,6 +1,6 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
+- [DEMO LINK](https://OlegRak86.github.io/layout_moyo-header/)
 - [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
```

Comment:
you should add the link to the test report here and write it in the PR description. Have you tried to solve this problem (asked in the chat, etc.)?
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/6457a5e8-fdd8-4bcb-bd51-736851c73ae2)

Reply:
I didn't write in the chat